src/Raven.Client/Documents/Attachments/RetiredAttachmentFlags.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Raven.Client.Documents.Attachments
+{
+    [Flags]
+    public enum RetiredAttachmentFlags
+    {
+        None = 0,
+        Retired = 0x1
+    }
+
+    internal static class EnumExtensions
```

Comment:
We have *3* separate `EnumExtensions` classes?

Reply:
This is a leftover I will remove `EnumExtensions`